libs/proto/scheduler.proto

```diff
@@ -222,6 +222,11 @@ message RegistrationResponse {
 	// The private key that was used to create the certificate with. This key
 	// is used to sign the requests to the node.
 	string device_key = 2;
+
+	// A master rune that is returned if the device that is registered has its
+	// own signer. The signer is necessary as the response is intercepted on the
+	// client side and appends the rune to the registratrion response.
+	string rune = 3;
```

Comment:
Ah, this is where we have the divergence we discussed between scheduler <-> client comms and gl-client <-> caller. The rune isn't returned by the scheduler, it is being backfilled by the signer when the result comes back in the same way the CSR -> CRT backfilling works already :+1:

Reply:
Exactly :)